api/src/pcapi/core/providers/api.py

```diff
@@ -127,11 +133,12 @@ def update_venue_provider(
         update_venue_synchronized_offers_active_status_job.delay(
             venue_provider.venueId, venue_provider.providerId, venue_provider.isActive
         )
-        logger.info(
+        logger.info(  # type: ignore [call-arg]
             "Updated VenueProvider %s isActive attribut to %s",
             venue_provider.id,
             venue_provider.isActive,
             extra={"is_active": venue_provider.isActive, "venue_provider_id": venue_provider.id},
+            technical_message_id="offer.sync.updated",
```

Comment:
(question / suggestion) Est-ce que cela suffira à la data de savoir qu'il y a eu un changement sur la synchronisation (sans savoir si c'est une activation ou une désactivation) ? On pourrait être plus précis en conditionnant le message selon l'action : 

```suggestion
            technical_message_id="offer.sync.activated" if venue_provider.isActive else "offer.sync.deactivated",
```